requirements.txt

```diff
@@ -2,4 +2,4 @@ dj-database-url==0.4.0
 Django==1.9.2
 gunicorn==19.4.5
 psycopg2==2.6.1
-whitenoise==2.0.6
+whitenoise==2.0.6
```

Comment:
Oh I wasn't complaining. I was just curious about it. Looks interesting. Can talk about it in the next meetup.